### ISSUE TITLE
App Review & App Downloads params

### ISSUE THREAD

**Issue by oorlov** [onepf/OpenAEP#3]
Current: [App Downloads](https://github.com/onepf/OpenAEP/blob/master/specification/openaep_spec_1_0.md#downloads) 
and [App Reviews](https://github.com/onepf/OpenAEP/blob/master/specification/openaep_spec_1_0.md#app-reviews)

Date:
- from [YYYY-MM-DD]
- to [YYYY-MM-DD]

both optional?

**Comment by jesper-sjovall** [comment]
Yes, Both date format, from and to, and I also suggests a 'limit' value for limit the result to a given number and in this case return the latest/newest of the reviews/posts.
This will get a impact of the natural sort order must be latest first and oldest last, which I think is fine for the must part of uses.

Examples: '
- Will return the 3 latest review for this apps from the store.
  https://www.sourceappstore.com/openaep/appreviews?package=com.softspb.flashcards.sv&limit=3
- Will return reviews in date rang 2013-10-15  to 2013-10-20 but never more then 100 reviews.
  https://www.sourceappstore.com/openaep/appreviews?package=com.softspb.flashcards.sv&limit=100&from=2013-10-15&to=2013-10-20

**Comment by jesper-sjovall** [comment]
As suggest from https://github.com/onepf/OpenAEP/issues/2, I also want to add country-code which I think will be must useful for only get reviews in a language that is supported by the store. 
For example, on a Swedish appstore this is no to little use of reviews written in Arabic

**Comment by oorlov** [comment]
Updated version of [downloads](https://github.com/onepf/OpenAEP/blob/master/specification/openaep_spec_1_0.md#downloads) :

``` ruby
https://www.sourceappstore.com/openaep/appreviews
?package=com.softspb.flashcards.sv - single parameter
[&datefrom=YYYY-MM-DD] - optional
[&dateto=YYYY-MM-DD] - optional
[&country=US] - optional
[&limit=100 &offsettoken= 500] - optional pair
```

Do paginate output there should be `offset` param used together with `limit`. And we need to return it in output like this

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<downloads version="1" offset=""> <!-- encoded value of request params, db state and offset -->
  <download>
...
```

is it simple to implement on backend?

**Comment by jesper-sjovall** [comment]
Use

```
[&limit=100] - optional
[&offset=7j8ad9go] - optional
```

and not 

```
[&limit=100 &offsettoken= 500] - optional pair
```

as in Purchases case https://github.com/onepf/OpenAEP/issues/2#issuecomment-26706540
For get a equal interface for all requests that will do the interface more easy to understand for everybody.

I think this is simple to implement on backend (for most of all systems), a loot of this will map very good to querys on for a Relational database as SQL-database.

**Comment by oorlov** [comment]
OK, lets update spec

**Comment by jesper-sjovall** [comment]
Ok, Update and I will look at it again.
